cluster-autoscaler/cloudprovider/aws/aws_manager.go

```diff
@@ -293,6 +304,28 @@ func extractLabelsFromAsg(tags []*autoscaling.TagDescription) map[string]string
 	return result
 }
 
+func extractResourceAllocatableFromAsg(tags []*autoscaling.TagDescription) map[string]*resource.Quantity {
```

Comment:
we already have one extractLabelsFromAsg method, is it possible to reuse method?

Reply:
I didn't do it, because `extractTaintsFromAsg` and `extractLabelsFromAsg` were already different functions. So I sticked with what was already there and made `extractResourceAllocatableFromAsg` separate.

I could go for a function with the signature like 

```
extractLabelsAndResourcesFromAsg(tags []*autoscaling.TagDescription) (map[string]string,  map[string]*resource.Quantity)
```
Although the `And` in a function name is already a red flag to me.

Another option would be to extract the parsing of the ASG labels, but that would only save us like 3 lines in the main function body (because we would still need to loop), so I don't think it is really worth it.

Which would you prefer?

Reply:
Make sense. Let's keep current signature. I will find opportunity to refactor all into one method later.